src/include/btmem.h

```diff
@@ -223,7 +223,7 @@ struct __wt_ovfl_reuse {
 #endif
 #define	WT_LAS_CONFIG							\
     "key_format=" WT_UNCHECKED_STRING(QIQu)				\
-    ",value_format=" WT_UNCHECKED_STRING(QuBu)				\
+    ",value_format=" WT_UNCHECKED_STRING(QuBuB)				\
```

Comment:
To discuss: Is there any value in having the schema as "QuBBu" instead of the one I have in the change now.

Reply:
The encoding is a little more efficient if the last field is a 'u' -- we can save storing the length in that case and just treat the rest of the bytes as the data for a `WT_ITEM`.